### DISCUSSION TITLE
Easy to way to make changes to pseudo elements

### DISCUSSION THREAD

**Discussion by MariuzM** [mantinedev/mantine#5822] [Q&A]
I have a Select component how can i change its `Focus border color`, and `hover background color`? i can't use modules because i have JS style i need insert into its color

**Comment by MariuzM** [comment]
I have tried this and able to change border but thats it, i can't find easy way to target its contents 

<img width="438" alt="image" src="https://github.com/mantinedev/mantine/assets/36937039/29bf7855-fb83-445a-abe0-645507782843">

Also MultiSelect does not seem to work for this change

**Reply by tombohub** [reply]
what do you mean by pseudo elements? `:focus` here is pseudo-class representing element state. Do you mean state?

Also select is combined component, which parts of select do you wanna change the focus border and bg color: https://mantine.dev/core/select/#styles-api

**Comment by rtivital** [comment]
https://help.mantine.dev/q/dynamic-css-styles

**Reply by MariuzM** [reply]
I got the emotion working just needed the right focus selector.

That said, i still think doing css modules is more of a workaround and code will look more complex vs emotion route. 

<img width="367" alt="image" src="https://github.com/mantinedev/mantine/assets/36937039/20d87494-dd41-40cd-a7dd-4d1eec1779ff">

But i find using these props is really nice that targets specific component, only thing is missing is Pseudo selection to change their colors 

<img width="558" alt="image" src="https://github.com/mantinedev/mantine/assets/36937039/18266bff-605d-4b22-8c63-958c7b5001fe">

**Reply by tombohub** [reply]
I don't think it's possible to do pseudo classes in inline styles. You have to use emotion or some other solution
